wgpu/src/backend/web.rs

```diff
@@ -1099,7 +1099,7 @@ impl crate::context::Context for Context {
         _adapter_data: &Self::AdapterData,
         format: wgt::TextureFormat,
     ) -> wgt::TextureFormatFeatures {
-        format.guaranteed_format_features()
+        format.guaranteed_format_features(wgt::Features::empty())
```

Comment:
I think we should pass the actual adapter features here or else validation will fail even if the feature is actually supported.